src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/ValidatorFactoryFactory.cs

```diff
@@ -22,35 +17,9 @@ public ValidatorFactoryFactory(ValidatorConfiguration validatorConfig, ILoggerFa
             _loggerFactory = loggerFactory ?? throw new ArgumentNullException(nameof(loggerFactory));
         }
 
-        public ValidatorFactory Create(string galleryUrl, string indexUrl)
+        public ValidatorFactory Create()
```

Comment:
Because the resource provider logic is no longer here, this class is pretty useless. I'd get rid of it and then call `ValidatorFactory` constructor in `PackageValidatorFactory.Create` directly.